autoscheduler/frontend/src/components/HelpText.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+
+const style = {
+  height: 350,
+  width: 650
+}
+
+const centerText = {
+  flex: 1,
+  justifyContent: 'center',
+  alignItems: 'center'
+}
+
+const HelpText: React.SFC = function App() {
```

Comment:
Typography doesn't seem to be working in this component, but it works in the Buttons. Do you need to use the material-ui `<Typography>` component to fix it?

Reply:
Yeah, every piece of text that uses typography needs to use the `Typography` component